ql/time/calendar.cpp

```diff
@@ -197,7 +197,7 @@ namespace QuantLib {
     }
 
     Day Calendar::WesternImpl::easterMonday(Year y) {
-        static const Day EasterMonday[] = {
+        static const unsigned char EasterMonday[] = {
```

Comment:
Reason for this?  The method returns a `Day`...

Reply:
The values all fit in a byte, so this saves space in the table. 3 bytes per entry, so about 1800 bytes for the two tables.

I thought it's a bit wasteful to store all these zeros in the data segment, but I can revert back if you think it's not worth it.

Reply:
My doubt is whether this forces a conversion from unsigned char to int each time the method returns.

Reply:
The conversion just gets folded into the mov instruction (`movzx` instead of plain `mov`): https://godbolt.org/z/5bf3PMaGE
They are the same speed when everything is in cache, but with the smaller tables the probability of being in cache is higher.